libqtile/widget/volume.py

```diff
@@ -214,21 +217,20 @@ def get_volume(self):
 
             mixer_out = subprocess.getoutput(get_volume_cmd)
         except subprocess.CalledProcessError:
-            return -1
+            return -1, 0
```

Comment:
Can you switch to True/False here? Other than that looks good to me.

Reply:
What do you mean exactly? Do you want to use True/False in self.mute instead of 1/0?

Reply:
yep, exactly

Reply:
Done